### PR TITLE
Start building on Ethereum section

### PR DESCRIPTION
**Pull request by andrealbiac**
Merged the SRE section (`start-building`) and the challenge card section (`test-your-knowledge`):
Captura de pantalla 2024-09-28 a las 16 32 29

into a single section that shows/suggests the journey of a dev in Buidlguidl, from SRE to batches and finally to grants:
Captura de pantalla 2024-09-28 a las 16 04 53

- For that I created a new card module (`JourneyCard`) with new content, and deleted the older `ChallengeCard.tsx` cause we're not using it anymore. 
- Also replaced the dice icon for the spaceship, and deleted unused images.

-------------------------------------------------
I do need help with one little thing: I can't manage to make the **bottom border** of this floor illustration clean, like it was before ⬇️ without showing the white line below it. 
Captura de pantalla 2024-09-28 a las 16 04 58

Tried tweaking the bottom margin and padding but it's not working. Maybe changing to `absolute` position would be a way but not sure if we want to do that?

-------------------------------------------------
I believe I'm not missing anything but let me know if you see something weird or have any feedback on the design as well!

Also, @austintgriffith the Batches link is currently pointing to https://batches.buidlguidl.com, not sure if that was the idea.

Thank you 🤗

### CODE REVIEW

**Comment by Pabl0cks** [comment]
> I do need help with one little thing: I can't manage to make the bottom border of this floor illustration clean, like it was before ⬇️ without showing the white line below it.

Moved the sre-path background to the _Scaffold ETH 2_ section in https://github.com/BuidlGuidl/buidlguidl.com/pull/41/commits/0d89ae89554f7949c6c5de60681faee6c7ec76d7 and played with negative margin a bit. Feel free to adjust spacing 🙏

File: packages/nextjs/pages/index.tsx [hunk 1]
```diff
@@ -48,76 +48,48 @@ const Home: NextPage<{
...
           <div className="flex gap-8 flex-wrap lg:flex-nowrap justify-center lg:max-w-5xl">
             {/* Card */}
             <Card
-              num={4}
-              name="Build a DEX"
+              name="Speedrun Ethereum"
               src="/assets/chall-dex.png"
-              description="💵 Build an exchange that swaps ETH to tokens and tokens to ETH."
+              description="Learn the most important concepts and level up your skills."
               link="https://github.com/scaffold-eth/se-2-challenges/tree/challenge-4-dex"
```
Comment (Pabl0cks) [inline packages/nextjs/pages/index.tsx:72]: We're still showing dex4 image and pointing to the dex 4 repo, is it ok?. Should we at least link to https://speedrunethereum.com/?

**Comment by Pabl0cks** [comment]
> Also, @austintgriffith the Batches link is currently pointing to https://batches.buidlguidl.com/, not sure if that was the idea.

Maybe we would point to the page that @portdeveloper is building? If so maybe we could add his page to this PR, since there isn't a lot of code changes 🙏

File: packages/nextjs/pages/index.tsx [hunk 2]
```diff
@@ -48,76 +48,48 @@ const Home: NextPage<{
...
             <Card
-              num={6}
-              name="Multisig Wallet"
-              src="/assets/chall-multisig.png"
-              description="👩‍👩‍👧‍👧 Using a smart contract as a wallet we can secure assets by requiring multiple accounts to 'vote' on transactions."
-              link="https://github.com/scaffold-eth/se-2-challenges/tree/challenge-6-multisig"
+              name="Small Grants"
+              src="/assets/bg-grants-card.png"
+              description="BuidlGuidl can sponsor up to 1 ETH to build your idea. Funding meaningful projects across the ecosystem."
+              link="https://grants-bg.vercel.app"
```
Comment (Pabl0cks) [inline packages/nextjs/pages/index.tsx:84]: We can point to: https://grants.buidlguidl.com/

File: packages/nextjs/components/JourneyCard.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+import TrackedLink from "~~/components/TrackedLink";
+
+/**
+ * Site header
+ */
+export const Card = ({
```
Comment (Pabl0cks) [inline packages/nextjs/components/JourneyCard.tsx:7]: Here we can name it as `JourneyCard` and update the import and usage in `index.tsx`, so it is more coherent with the other card naming (BuildCard and LearnMoreCard)

**Review by Pabl0cks — CHANGES_REQUESTED** [review]
Nice job @andrealbiac !! Added some comments, will finish the review tonight 🙏

File: packages/nextjs/pages/index.tsx (hunk 1 above)
Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx:72]: Hey! Good point, we should definitely change the link to the SRE site, yes. 
About the image I'm not sure, I like that it shows a challenge title but let me get back to you on this one!

Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx:72]: Also, do you see this image like this? 
Captura de pantalla 2024-09-30 a las 18 01 30

I thought I fixed that but now it's off again

**Comment by andrealbiac** [comment]
Thanks @Pabl0cks  🙌🏻

I'll see how the SRE card it looks with a more general image. Otherwise I changed the card name as well, and the sre-background looks great! Thanks for the help 🫶🏻

And merging ports PR here sounds good! Not sure how that works of if I need to do something so keep me updated!

File: packages/nextjs/components/JourneyCard.tsx (hunk 3 above)
Comment (andrealbiac) [reply packages/nextjs/components/JourneyCard.tsx:7]: Just added the fix but I'm getting this error, so I think I did something wrong 🤔
Changed the name in `index.tsx` and the `export const` in `JourneyCard`. Might be missing something

Captura de pantalla 2024-09-30 a las 18 14 29

Comment (Pabl0cks) [reply packages/nextjs/components/JourneyCard.tsx:7]: In `index.tsx` did you update both the import and the <Card> elements?

File: packages/nextjs/pages/index.tsx (hunk 2 above)
Comment (Pabl0cks) [reply packages/nextjs/pages/index.tsx:84]: Did you push it to this branch?

File: packages/nextjs/components/JourneyCard.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+import TrackedLink from "~~/components/TrackedLink";
+
+/**
+ * Site header
+ */
```
Comment (Pabl0cks) [inline packages/nextjs/components/JourneyCard.tsx:6]: You can delete this comment

File: packages/nextjs/components/JourneyCard.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  link: string;
+}) => {
+  return (
+    <TrackedLink
+      id={name}
+      href={link}
+      className="flex flex-col lg:w-1/3 max-w-xs shadow-lg bg-skin rounded-[46px] mt-4"
+    >
+      <div className="flex w-full h-[200px] relative">
+        <Image src={src} fill alt="Dex Challenge" className="w-full object-cover rounded-t-[46px]" />
```
Comment (Pabl0cks) [inline packages/nextjs/components/JourneyCard.tsx:25]: The alt was also hardcoded in ChallengeCard component, instead we can do like in BuidlCard and set:
`alt={name}`

**Comment by Pabl0cks** [comment]
Added a couple more comments, let me know if you need help with any of the changes 🙌

File: packages/nextjs/pages/index.tsx (hunk 2 above)
Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx:84]: Yes! You can find it above these comments in this thread, right before your commit

File: packages/nextjs/pages/index.tsx (hunk 1 above)
Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx:72]: Nevermind, it's fixed now!

File: packages/nextjs/components/JourneyCard.tsx (hunk 3 above)
Comment (andrealbiac) [reply packages/nextjs/components/JourneyCard.tsx:7]: I did, but I forgot to save the JourneyCard.tsx file 😅 okay I got it now!

**Comment by andrealbiac** [comment]
Fixed all the JourneyCard comments, thanks! 

I have doubts about the SRE image, cause it's more coherent with the other cards if we change it, but also too repetitive with the girl running twice (with the bottom background image). 
But I wouldn't remove the bottom one cause it's fun and more dynamic than other sections.

So maybe we keep the image of the challenge as it is?
What do you think?

Captura de pantalla 2024-10-01 a las 12 48 02
Captura de pantalla 2024-10-01 a las 12 53 55

File: packages/nextjs/pages/index.tsx (hunk 2 above)
Comment (andrealbiac) [reply packages/nextjs/pages/index.tsx]: Oh wait! I read it wrong 😅 I thought we were only talking about batches all the time, sorry! 

Just updated it now in a commit 👌🏻

**Comment by Pabl0cks** [comment]
I think we should reduce the image sizes a bit for the /batches page @portdeveloper , they feel a bit too heavy! 

I usually use tools like tinypng to do that automatically, maybe @andrealbiac can help us with that too 🙏 (tinypng is not reducing much the castle and planet images, the other 2 are being reduced ~70% which is fine)

File: packages/nextjs/pages/batches/index.tsx [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+  },
+  {
+    name: "Batch #9",
+    participants: 11,
+    startDate: "Sep 14",
+    batchPageLink: "https://batch9.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch9.buidlguidl.com",
+  },
+];
+
```
Comment (carletex) [inline packages/nextjs/pages/batches/index.tsx:74]: Note: @phipsae is working on getting this data into the BG firebase database, so soon we'd have an API endpoint to get all the batch data.

**Comment by andrealbiac** [comment]
> I think we should reduce the image sizes of the pngs a bit for the /batches

Yeah sorry about that! I think I already used an image compressor but maybe it wasn't enough. I'll get the one you use as well for next times

**Review by Pabl0cks — APPROVED** [review]
TYSM everyone! Good job.

Let's have the changes merged for the live review next week.

File: packages/nextjs/pages/batches/index.tsx (hunk 6 above)
Comment (Pabl0cks) [reply packages/nextjs/pages/batches/index.tsx]: Creating an issue to change the hardcoded data to the API endpoint whenever is ready 🙏

**Pull request merged**